src/core/truthTable/truth_table.cpp

```diff
@@ -11,19 +11,21 @@ namespace syrec {
             if (cube[pos].has_value()) {
                 continue;
             }
-
             // recursively compute all the complete cubes for the zero case
-            auto zero         = Cube(cube);
-            zero[pos]         = false;
+            Cube zero(cube);
+            zero.at(pos) = false;
```

Comment:
Is there a reason for these lines being changed?

Reply:
zero[pos] was never modifying the element in the specific position since [] operator returns a const. As a result, it ended up being a infinite recursion. Since vectors have an at method, I thought I could define an at method as well